tests/src/Functional/CiviCrmTestBase.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\Tests\webform_civicrm\Functional;
+
+use Drupal\Core\Database\Database;
+use Drupal\Tests\BrowserTestBase;
+
+// Requires patching for civicrm-core.
+// @see https://github.com/civicrm/civicrm-core/pull/18843
+// @see https://lab.civicrm.org/dev/core/-/issues/2140
+// @todo move into civicrm-drupal-8 package.
```

Comment:
I wasn't sure if the main module would want the patch in its `composer.json` and all of the `@todo` items. I figure easier to keep here and shuffle around once CiviCRM gets a release. Then move test bases into CiviCRM D8 module. Then update Webform CiviCRM